plugins/Annotations/templates/getEvolutionIcons.twig

```diff
@@ -5,7 +5,7 @@
         <span data-date="{{ date }}" data-count="{{ counts.count }}" data-starred="{{ counts.starred }}"
               {% if counts.count == 0 %}title="{{ 'Annotations_AddAnnotationsFor'|translate(date) }}"
               {% elseif counts.count == 1 %}title="{{ 'Annotations_AnnotationOnDate'|translate(date,
-              counts.note)|raw }}
+              counts.note)|e('html_attr') }}
```

Comment:
Theoretically it would be enough to only escape `counts.note`. That would allow the translation to contain html tags (which it currently doesn't have)